src/axes/numericAxis.ts

```diff
@@ -262,6 +278,31 @@ export module Axes {
       }
     }
 
+    /**
+     * Gets the approximate text width setting.
+     *
+     * @returns {boolean} The current text width approximation setting.
+     */
+    public usesTextWidthApproximation(): boolean;
+    /**
+     * Sets the approximate text width setting. Approximating text width
+     * measurements can drastically speed up plot rendering, but the plot may
+     * have extra white space that would be eliminated by exact measurements.
+     * Additionally, very abnormal fonts may not approximate reasonably.
+     *
+     * @param {boolean} The new text width approximation setting.
+     * @returns {Plottable.Axes.Numeric} The calling Plottable.Axes.Numeric.
```

Comment:
Sorry I think I was unclear...

Can you replace with
`{Axes.Numeric} The calling Axes.Numeric.`